roles/backup/tasks/remove_management_pod.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: Delete any existing management pod
+  k8s:
+    name: "{{ ansible_operator_meta.name }}-backup-manager"
+    kind: Pod
```

Comment:
Maybe not to handle in this PR but as a follow
We should always define the `api_version` parameter with the k8s to avoid some issue with the default api_version value (see some awx-operator about this)